helm/examples/gke-nvme-ssd-block.yaml

```diff
@@ -0,0 +1,4 @@
+classes:
```

Comment:
Maybe add a "raid" suffix to this file, so it's clear it's the configuration to use with raid.

Reply:
Done